app/src/androidTest/java/com/github/h3lp3rs/h3lp/forum/ForumAnswersActivityTest.kt

```diff
@@ -73,11 +89,37 @@ class ForumAnswersActivityTest {
     }
 
     @Test
-    fun addNewAnswerWorks() {
-        onView(withId(R.id.text_view_enter_answer)).perform(replaceText(ANSWER_TEST))
+    fun addNewAnswerWorksForVerifiedProUser() {
+        val proUser = ProUser(USER_ID_NAME, USER_ID_NAME, "", "", "", "", "")
+        proUsersDb.setObject(USER_ID_NAME, ProUser::class.java, proUser)
 
-        onView(withId(R.id.add_answer_button)).perform(click())
+        launchActivity()
 
+        onView(withId(R.id.text_view_enter_answer)).perform(replaceText(ANSWER_TEST))
+        Thread.sleep(1000)
```

Comment:
This is a very bad practice, but I use it as well since kotlin doesn't support delayed future execution :(
But maybe put a comment and/or add the delay in a constant object.

Reply:
Ah oops, that's actually not needed. It was just for me to check something lol. It will be removed